### PR TITLE
chore: transfer contents of layer5-ng

### PR DESCRIPTION
**Pull request by Tanuj22**
**Description**
Transfers the contents of `layer5-ng` repo.

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: layer5-ng/.firebaserc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (leecalcote) [inline layer5-ng/.firebaserc:1]: @Tanuj22 we might be able to be rid of this file, I suppose. What do you think?

**Comment by leecalcote** [comment]
@Tanuj22 perhaps, you can introduce the layer5-ng site on today's Dev call.

**Comment by Tanuj22** [comment]
> @Tanuj22 perhaps, you can introduce the layer5-ng site on today's Dev call.

Sorry @leecalcote . I completely missed this message

File: layer5-ng/.firebaserc (hunk 1 above)
Comment (Tanuj22) [reply layer5-ng/.firebaserc:1]: Yup. Will remove in the next PR

**Pull request merged**